qa/L0_backend_python/python_test.py

```diff
@@ -246,76 +254,92 @@ def test_async_infer(self):
+                            output_data, input_datas[i]
+                        ),
+                    )
 
                 # Make sure the requests ran in parallel.
                 stats = client.get_inference_statistics(model_name)
-                test_cond = (len(stats['model_stats']) != 1) or (
-                    stats['model_stats'][0]['name'] != model_name)
+                test_cond = (len(stats["model_stats"]) != 1) or (
+                    stats["model_stats"][0]["name"] != model_name
+                )
+                self.assertFalse(
+                    test_cond, "error: expected statistics for {}".format(model_name)
+                )
+
+                stat = stats["model_stats"][0]
                 self.assertFalse(
-                    test_cond,
-                    "error: expected statistics for {}".format(model_name))
-
-                stat = stats['model_stats'][0]
-                self.assertFalse((stat['inference_count'] != 8) or (
-                    stat['execution_count'] != 1
-                ), "error: expected execution_count == 1 and inference_count == 8, got {} and {}"
-                                 .format(stat['execution_count'],
-                                         stat['inference_count']))
-                batch_stat = stat['batch_stats'][0]
+                    (stat["inference_count"] != 8) or (stat["execution_count"] != 1),
+                    "error: expected execution_count == 1 and inference_count == 8, got {} and {}".format(
+                        stat["execution_count"], stat["inference_count"]
+                    ),
+                )
+                batch_stat = stat["batch_stats"][0]
                 self.assertFalse(
-                    batch_stat['batch_size'] != 8,
-                    f"error: expected batch_size == 8, got {batch_stat['batch_size']}"
+                    batch_stat["batch_size"] != 8,
+                    f"error: expected batch_size == 8, got {batch_stat['batch_size']}",
                 )
                 # Check metrics to make sure they are reported correctly
-                metrics = httpreq.get('http://localhost:8002/metrics')
+                metrics = httpreq.get("http://localhost:8002/metrics")
                 print(metrics.text)
 
-                success_str = 'nv_inference_request_success{model="identity_uint8",version="1"}'
-                infer_count_str = 'nv_inference_count{model="identity_uint8",version="1"}'
-                infer_exec_str = 'nv_inference_exec_count{model="identity_uint8",version="1"}'
+                success_str = (
+                    'nv_inference_request_success{model="identity_uint8",version="1"}'
+                )
+                infer_count_str = (
+                    'nv_inference_count{model="identity_uint8",version="1"}'
+                )
+                infer_exec_str = (
+                    'nv_inference_exec_count{model="identity_uint8",version="1"}'
+                )
```

Comment:
TIL that `(element) == element: T` but `(element,) == element: Tuple[T]` CC @dyastremsky @Tabrizian for the fun fact:
```python
# I thought this would be a tuple of one element, but is a string
>>> x = ("hi")
>>> x
'hi'
>>> type(x)
<class 'str'>      # thought this would be a tuple or set

# This is a tuple
>>> x = ("hi",)
>>> x
('hi',)
>>> type(x)
<class 'tuple'>

# Though this is a list rather than a string, not the same rules as tuple
>>> x = ["hi"]
>>> x
['hi']
>>> type(x)
<class 'list'>
```